packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -130,6 +176,66 @@ contract SecurityStore is DataStore
         }
         w.guardians.pop();
         delete w.guardianIdx[guardian];
+        delete w.guardianRemovalEffectiveTime[guardian];
```

Comment:
Because here we just cancel add a guardian, so I think we should also delete it from guardianEffectiveTime.
delete w.guardianEffectiveTime[guardian];